src/chrome/content/rules/Nature.com.xml

```diff
@@ -0,0 +1,152 @@
+<!--
+	Time out:
+		nature.com
+		bts-wp.nature.com
+		darwin.nature.com
+		stingray.nature.com
+
+	Refused:
+		api.nature.com
+		blogs.nature.com
+		www.blogs.nature.com
+		biotech.nature.com
+		cellbio.nature.com
+		data.nature.com
+		demo.nature.com
+		deutsche.nature.com
+		eagleray.nature.com
+		bounce.ealert.nature.com
+		open.ealert.nature.com
+		reply.ealert.nature.com
+		external.nature.com
+		genetics.nature.com
+		guide.nature.com
+		www.guide.nature.com
+		immunology.nature.com
+		www.immunology.nature.com
+		mailart.nature.com
+		materials.nature.com
+		medecine.nature.com
+		www.medecine.nature.com
+		medicalresearch.nature.com
+		naturejobs.nature.com
+		network.nature.com
+		neurosci.nature.com
+		npg.nature.com
+		ns.nature.com
+		palgrave.nature.com
+		physics.nature.com
+		platformsupport.nature.com
+		precedings.nature.com
+		preceedings.nature.com
+		press.nature.com
+		proceedings.nature.com
+		protocols.nature.com
+		reports.nature.com
+		search.nature.com
+		split.nature.com
+		structbio.nature.com
+		support.nature.com
+		xml.nature.com
+		xps.nature.com
```

Comment:
Given the message over `http`:

> my.nature.com has been discontinued

> Following a beta pilot, my.nature.com discontinued service on April 14th, 2014.

> We apologize for any inconvenience caused.

is it really necessary to include it?